.github/workflows/golangci-lint.yaml

```diff
@@ -11,19 +11,15 @@ jobs:
   golangci-lint:
     name: golangci-lint
     runs-on: ubuntu-latest
-    strategy:
-      fail-fast: false
-      matrix:
-        go: ["1.20", "1.21"]
     steps:
-      - uses: actions/setup-go@v2
-        with:
-          go-version: ${{ matrix.go }}
+      - uses: actions/checkout@v4
 
-      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v5
+        with:
+            go-version-file: go.mod
```

Comment:
Is there a reason for the step order change?

Reply:
Same as https://github.com/red-hat-storage/ocs-client-operator/pull/92#discussion_r1512926724